schemas/common/beacon.schema.json

```diff
@@ -0,0 +1,60 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/beacon",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Beacon",
+  "type": "object",
+  "description":
+    "Beacon, a wireless device that communicates identity information to mobile applications as mobile devices come within range.",
+  "definitions": {
+    "beacon": {
+      "properties": {
+        "xdm:proximityUUID": {
```

Comment:
Is it required that this field have the UUID format - https://en.wikipedia.org/wiki/Universally_unique_identifier - or is this just an arbitrary, unique string controlled by the customer?  If it's not required to be a UUID I would rename this to just ID.

Reply:
Yes, this is the expected format. A beacon has a UUID that it is programmed to emit. It is confusingly called a "Proximity UUID"